go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.0.0
 	github.com/aws/aws-sdk-go v1.44.325
 	github.com/bshuster-repo/logrus-logstash-hook v1.0.0
+	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
```

Comment:
Noticed one thing here. Can we please ping this dependency to the latest available [release](https://github.com/coreos/go-systemd/releases) which seems to be `v22.5.0`. This one seems 4 years old 😅

Reply:
Huh...that's what I got by default.  I've updated it now.